src/lib/flow/codegen/blocklayer/prelude.ts

```
@@ -0,0 +1,145 @@
+export const applyPrelude = (prelude: IPreludeData, bodyNodes: Node[]): AppliedPrelude => {
+	switch (prelude.type) {
+		case IPreludeTypeEnum.Command:
+			const argTypeToDiscord = {
+				[ICommandArgumentType.String]: 'String',
+				[ICommandArgumentType.Integer]: 'Integer',
+				[ICommandArgumentType.Boolean]: 'Boolean',
+				[ICommandArgumentType.User]: 'User',
+				[ICommandArgumentType.Channel]: 'Channel',
+				[ICommandArgumentType.Role]: 'Role',
+				[ICommandArgumentType.Member]: 'Member'
+			};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)